retriever/bitbucketretriever/retriever.go

```diff
@@ -0,0 +1,103 @@
+qpackage bitbucketretriever
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/thomaspoignant/go-feature-flag/internal"
+	"github.com/thomaspoignant/go-feature-flag/retriever/shared"
+)
+
+type Retriever struct {
+	RepositorySlug     string
+	Workspace          string
```

Comment:
**suggestion:** For `Github` and `Gitlab` retrievers, the repository slug is the concatenation of the workspace and the project name (ie: `my-workspace/go-feature-flag`). We should have something similar here and remove the the `Workspace` property.

```suggestion
	RepositorySlug     string
```